### DISCUSSION TITLE
Button to quickly show the generated JS output of an F# Fable file

### DISCUSSION THREAD

**Discussion by Darkle** [ionide/ionide-vscode-fsharp#1880] [Ideas]
Hi, when using Rescript in VScode, the Rescript VSCode extension has a neat feature where you can click a button and it will show you the Javascript file that was generated for the Rescript file you have open. I think this would be a good feature to have in the Ionide extension too, as when using F# with Fable, you often want to check the output is right. 

Screenshots of how it looks in Rescript extension:
Before clicked on button:
![2023-05-30_16-13](https://github.com/ionide/ionide-vscode-fsharp/assets/6868833/2bce5d36-e1c2-46c2-b8d9-e7a85507569a)

After clicked on button:
![2023-05-30_16-14](https://github.com/ionide/ionide-vscode-fsharp/assets/6868833/0cecc71c-5315-435e-bdc8-cc93365b3bc5)

**Comment by MangelMaxime** [comment]
The difficulty with this feature is that Fable don't always generate the files alongside the source files.

Indeed, when using Fable user can specify an output directory. And I am not sure there is an elegant way of detecting these files.

Should we cover only the case where we detect `source.fs.js` files alongside the originated `source.fs` file?
